src/common/lib/api.ts

```diff
@@ -111,9 +119,11 @@ export const setSetting = (setting: MessageSettingsSet["args"]["setting"]) =>
     setting,
   });
 export const removeAccount = (id: string) =>
+  // @Todo: maybe remove Promise.all
   Promise.all([
     msg.request("removeAccount", { id }),
-    removeAccountFromCache(id),
+    // @Todo: how to remove account from swr cache?
+    // @Todo swr dows say "mutate(key)" will mark the data as expired and trigger a refetch for the resource
```

Comment:
I don't think this is really necessary to delete it from the SWR cache since it's only stored for the duration of the session